### PR TITLE
changes to sheet class inheritance

### PR DESCRIPTION
**Pull request by ksobon**
### Purpose

Sheet class should probably inherit from the View class to match the Revit API inheritance structure. Also this solves the issue here: https://github.com/DynamoDS/DynamoRevit/issues/1662

capture

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] Snapshot of UI changes, if any.

### Reviewers

@mjkkirschner 

### FYIs

@mjkkirschner can you run some tests for this? Again, I can't really get RTF to work so can't verify this doesn't wreck anything. It should be fine but you never know.

### CODE REVIEW

File: src/Libraries/RevitNodes/Elements/Views/Sheet.cs [hunk 1]
```diff
@@ -324,93 +321,40 @@ public View[] Views
...
             if (views == null)
             {
-                throw new ArgumentNullException("views");
+                throw new ArgumentNullException(nameof(views));
             }
 
             if (views.Length == 0)
             {
-                throw new ArgumentException("Must supply more than 0 views");
+                throw new ArgumentException(Properties.Resources.Sheet_NoViewsError);
```
Comment (mjkkirschner) [inline src/Libraries/RevitNodes/Elements/Views/Sheet.cs]: 👍

**Comment by QilongTang** [comment]
LG to merge and Dynamo Team is waiting for RTF test results

**Comment by ksobon** [comment]
Should I be providing that, or you guys are going to run your own?

**Comment by QilongTang** [comment]
@ksobon We thought you are having trouble on RTF

**Comment by ksobon** [comment]
Having trouble to get that thing to return a results XML file (report). It runs, but I never know if my tests passed.

**Comment by QilongTang** [comment]
@ksobon I have run RTF for your branch and results looks good to me. Merging. Let me know if you feel comfortable cherry picking  this to R2018 and R2016 branch.

**Comment by QilongTang** [comment]
@ksobon After discussion with @Racel, looks like  you have some issue locally and I will go ahead and do the cherry-picking of this PR. I also have a small coming PR helping running RevitSystemTests locally, will ping you in the PR.

**Pull request merged**